sdk/src/client/api/address.rs

```diff
@@ -117,7 +139,31 @@ impl SecretManager {
         )
     }
 
-    /// Get a vector of EVM address strings
+    /// Generates a single EVM address hex string.
+    pub async fn generate_evm_address(
+        &self,
+        coin_type: u32,
+        account_index: u32,
+        address_index: u32,
+        options: impl Into<Option<GenerateAddressOptions>> + Send,
+    ) -> Result<String> {
+        Ok(prefix_hex::encode(
+            SecretManage::generate_evm_addresses(
+                self,
+                coin_type,
+                account_index,
+                address_index..address_index + 1,
+                options,
+            )
+            .await?
+            .pop()
+            // Panic: if the secret manager hasn't failed then there must be an address.
+            .unwrap()
```

Comment:
Just use `[0]` like you did in `generate_ed25519_address`, no need for that inconsistency

Reply:
Done